lib/everypolitician/popolo/person.rb

```diff
@@ -3,22 +3,64 @@ module Popolo
     class Person < Entity
       class Error < StandardError; end
 
-      attr_accessor :name, :email, :image, :gender, :birth_date, :death_date, :honorific_prefix, :honorific_suffix
+      def family_name
+        document[:family_name]
+      end
 
-      def links
-        document.fetch(:links, [])
+      def given_name
+        document[:given_name]
       end
 
-      def link(type)
-        links.find(-> { {} }) { |i| i[:note] == type }[:url]
+      def identifiers
+        document[:identifiers]
       end
 
-      def contact_details
-        document.fetch(:contact_details, [])
+      def images
+        document[:images]
```

Comment:
Things that return lists are going to need to a little more complex than this, I suspect.